src/datastore/postgres-store.ts

```diff
@@ -492,7 +492,7 @@ interface MempoolTxQueryResult {
   coinbase_payload?: Buffer;
 
   // sending abi in case tx is contract call
-  abi?: string;
+  abi: unknown | null;
```

Comment:
Note: using `unknown` to represent the JSON object here for lack of a better type. Another option is `Record<string, unknown>`, which typescript/linters often recommend (e.g. when trying to use the `Object` type).

We could also use `ClarityAbi` from stacks.js, although I'm somewhat wary of depending on an external lib's type in this area.

@rafaelcr I don't have strong feelings one way or the other, let me know what you think.

Reply:
I was curious about this yesterday too, actually, but I arrived at the same conclusion that `unknown` might be ok for now... I think it's safe, we can re-evaluate later if there's a better alternative